tuned/plugins/plugin_bootloader.py

```diff
@@ -51,6 +55,88 @@ def _get_config_options(cls):
 			"cmdline": None,
 		}
 
+	@staticmethod
+	def _options_to_dict(options, omit=""):
```

Comment:
There can be multiple kargs with same name (ie. options="A=B A B=C"), it can probably written differently, but I think this is more readable than some crazy one liner, but I can look into it

Reply:
OK, probably not worth to change. Could you add some comment with the example, e.g. transforming "A=B A B=C" to {"A": [None, "B"], "B": ["C"]} or similar comment.